GX-TimeFrequency/src/minimax_grids.F90

```diff
@@ -100,9 +75,9 @@ subroutine gx_minimax_grid(num_points, e_min, e_max, &
     real(kind=dp), dimension(:, :), allocatable       :: mat
     real(kind=dp), dimension(:, :), allocatable       :: tmp_cosft_wt, tmp_cosft_tw
 
-    my_bare_cossine_weights = .false.
-    if (present(bare_cossine_weights)) then
-      if(bare_cossine_weights) my_bare_cossine_weights = .true.
+    my_bare_cos_sin_weights = .false.
+    if (present(bare_cos_sin_weights)) then
+       my_bare_cos_sin_weights = .true.
```

Comment:
this is now wrong. Somebody could also pass bare_cos_sin_weight as .false. Please revert to the original code.

Reply:
Thanks for catching this one. I fixed it in [1943252](https://github.com/nomad-coe/greenX/pull/36/commits/194325236907e2d0220bc69ddccae576e9a9b890)